features/facebookImport/src/model/analyses/ministories/activities-analysis.js

```diff
@@ -16,29 +16,29 @@ export default class ActivitiesAnalysis extends RootAnalysis {
         return i18n.t("activitiesMiniStory:title");
     }
 
-    async analyze({ facebookAccount }) {
+    async analyze({ dataAccount }) {
```

Comment:
Why isn't it still a `facebookAccount`? From the perspective of the Facebook feature (where this code is), it definitely is, no?

Reply:
The issue that arises by keeping the naming with facebookAccount is that when `runAnalysis` from the poly-analysis package is called; it runs all the analyses it has been given by the feature and performs `analysis.analyze(enrichedData)`. The analysis on the feature end expects the enrichedData to contain a facebookAccount attribute whereas now the enrichedData contains a dataAccount attribute due to it being generated in poly-analysis which is shared. We could change `runAnalysis`  to accept another argument that signifies what type of dataAccount is expected by the analysers on the feature end, or we could keep it generic and work with dataAccount everywhere.

Reply:
Yeah, guess I missed that this is destructuring, so we'd have to explicitly rename it here - seems like overkill indeed. Then I'm fine with it as it is.